tests/utils.js

```diff
@@ -633,10 +633,10 @@ module.exports = {
 
   setProcessedSeqToNow: () => {
     return Promise.all([
-      sentinel.get('_local/sentinel-meta-data').catch(() => ({_id: '_local/sentinel-meta-data'})),
+      sentinel.get('_local/transitions-seq').catch(() => ({_id: '_local/transitions-seq'})),
```

Comment:
This appears to be called from the infodocs e2e test so it's probably more interested in the `background-seq` doc rather than the `transitions-seq`. Maybe it should update both of them? Or the function name should be changed to be more specific?

Reply:
Good catch, name changed to be more specific.

Reply:
(tbc the goal is to just stop transitions from noticing specific writes so we can test certain scenarios, so it's appropriate that it just cares about this seq, at least for now)